tests/jballerina-unit-test/src/test/resources/test-src/javainterop/basic/async_test.bal

```diff
@@ -11,6 +12,7 @@ public function countSlowly() returns int = @java:Method {
 
 public function main() {
     completeFutureMoreThanOnce();
+    runtime:sleep(1);
```

Comment:
Why do we need a `sleep()` call here?

Reply:
The test was intermittently failed. In this test we complete the future using multiple threads.
If thread 1 complete, thread 2 complete then the main will return with error. But sometimes main will return before thread 2 complete before error message in system out, so test will finish with an error given "error message not found". Added sleep to make sure main will not exit.